### PR TITLE
Accessor `get_size` and `get_range` fixes

### PR DESCRIPTION
**Pull request by ProGTX**
* https://github.com/KhronosGroup/SYCL-Docs/pull/31
* Removed `get_size` check for image accessors
* Check `get_range` for local accessors
* Check `get_range` for image accessors

### CODE REVIEW

**Comment by Ruyk** [comment]
SYCL Group: @keryell  to take a look

File: tests/accessor/accessor_api_local_common.h [hunk 1]
```diff
@@ -62,6 +62,24 @@ class check_local_accessor_api_methods {
   size_t count;
   size_t size;
```
Comment (keryell) [inline tests/accessor/accessor_api_local_common.h]: Are these 2 members used somewhere?

Comment (ProGTX) [reply tests/accessor/accessor_api_local_common.h]: Yes, when checking `get_count` and `get_size` later on

**Comment by bader** [comment]
@ProGTX, FYI. CTS still uses `get_size` method of image accessor in https://github.com/KhronosGroup/SYCL-CTS/blob/SYCL-1.2.1/master/tests/accessor/accessor_constructors_image_utility.h.

**Pull request merged**